src/containers/menu/Menu.tsx

```diff
@@ -38,7 +38,7 @@ const Menu = (props: MenuProps) => {
       position='bottomRight'
       renderTrigger={props.children}
       popupClassName={className}
-      zIndex={12}
+      zIndex={menu.type === 'notification' ? 10000 : 12}
```

Comment:
i dont think there are any other spots, but maybe worth looking into making sure modals show up on top of this

Reply:
They do! The popup up menu closes whenever an item is clicked. But even in the case that it's open when a modal appears the modal shows over it